storage/innobase/log/log0chkp.cc

```diff
@@ -1199,6 +1201,9 @@ void log_calc_max_ages(log_t &log) {
   /* Round limits to equal size of OS_FILE_LOG_BLOCK_SIZE, not to risk
   inproper lsn computations (we.g. value % OS_FILE_LOG_BLOCK_SIZE == 1) */
 
+  log.max_checkpoint_age.store(
```

Comment:
Why do we update the same var `log.max_checkpoint_age` within the same function twice (with unaligned and with aligned value)?

Reply:
Oh, you asked this question already ;) nice